### DISCUSSION TITLE
Allow Padding Control In Slides for Custom Style of Each Item

### DISCUSSION THREAD

**Discussion by ShantanuTonde** [elementor/elementor#15409] [Feature Request]
Slides have the standard settings for padding as global for all the items in that slider. I was trying to have different positions for content and buttons for each item as I had used custom images. But since the custom option in item style does not provide the padding setting, it is impossible to make it aesthetically pleasing. Available options of Horizontal Positioning, Vertical Positioning, and Text Alignment are not enough to make substantial changes.

**Comment by rami-elementor** [comment]
@ShantanuTonde Elementor has 7 slider / carousel widgets, which one are talking about?
